packages/create-block-interactive-template/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+const { join } = require( 'path' );
+
+module.exports = {
+	defaultValues: {
+		slug: 'example-interactive',
+		title: 'Example Interactive',
+		description: 'An interactive block with the Interactivity API',
+		dashicon: 'interactivity',
+		npmDependencies: [ '@wordpress/interactivity' ],
+		supports: {
+			interactivity: true,
+		},
+		render: 'file:./render.php',
+		viewScript: 'file:./view.js',
+	},
+	variants: {
+		basic: {},
+	},
+	pluginTemplatesPath: join( __dirname, 'plugin-templates' ),
+	blockTemplatesPath: join( __dirname, 'block-templates' ),
+	assetsPath: join( __dirname, 'assets' ),
```

Comment:
Is this needed? There's no `assets` folder…

Reply:
You're right. It's not needed. I've removed it 👍